proxysql-admin

```diff
@@ -1853,6 +1857,32 @@ function find_cluster_node()
   return 1
 }
 
+# Create query rules for given user. This is applicable only for singlewrite mode  
+# Globals:
+#   WRITER_HOSTGROUP_ID
+#
+# Arguments:
+#   None
```

Comment:
There is one argument, the username.

Reply:
Done 4ef41369f37bd111a34a5b6ebc8df25cbab7ea21